bin.js

```diff
@@ -17,6 +17,16 @@ if (arg('-h') || arg('--help')) {
     forceColor: arg('-c'),
     messageKey: messageKeyArg()
   })).pipe(process.stdout)
+  process.stdin.on('end', closeProcess)
```

Comment:
It seems to me you could just pass `process.exit` in both instances instead of creating a redundant `closeProcess` function.